meson.build

```diff
@@ -0,0 +1,53 @@
+project('netplan', 'c',
+        version: '0.104',
+        license: 'GPL3',
+        default_options: [
+            'c_std=c99',
+            'warning_level=1',
+            'werror=true',
+        ],
+        meson_version: '>= 0.61.0',
+)
+
+glib = dependency('glib-2.0')
+gio  = dependency('gio-2.0')
+yaml = dependency('yaml-0.1')
+uuid = dependency('uuid')
+libsystemd = dependency('libsystemd')
+
+systemd = dependency('systemd')
+completions = dependency('bash-completion')
+systemd_generator_dir = systemd.get_variable(pkgconfig: 'systemdsystemgeneratordir')
+systemd_unit_dir = systemd.get_variable(pkgconfig: 'systemdsystemunitdir')
+bash_completions_dir = completions.get_variable(pkgconfig: 'completionsdir', default_value: '/etc/bash_completion.d')
+
+pandoc = find_program('pandoc', required: false)
```

Comment:
You might want to use a meson_options.txt feature option for this. That would allow people to specify `-Ddoc=disabled` and force disable trying to detect pandoc or build anything with it.